[2022-03-24 08:48:18] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 08:48:33] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 08:52:18] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 08:52:33] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 08:56:19] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 08:56:33] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:00:19] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:00:34] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:04:19] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:04:34] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:08:20] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:08:35] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:12:20] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:12:35] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:16:21] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:16:36] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:20:21] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:20:36] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:24:22] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:24:37] [ERROR]: Protocol -> parse incoming packet error: return code unsuccessful: -10003 
[2022-03-24 09:28:04] [WARNING]: 恢复会话失败: exchange_emp failed: 16 , 尝试使用正常流程登录. 
[2022-03-24 09:28:06] [WARNING]: 账号已开启设备锁，请选择验证方式: 
[2022-03-24 09:28:06] [WARNING]: 1. 向手机 132******33 发送短信验证码 
[2022-03-24 09:28:06] [WARNING]: 2. 使用手机QQ扫码验证. 
[2022-03-24 09:28:06] [WARNING]: 请输入(1 - 2) (将在10秒后自动选择2)： 
[2022-03-24 09:28:08] [WARNING]: 请输入短信验证码： (Enter 提交) 
[2022-03-24 09:28:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 09:28:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 09:28:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 09:29:00] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":34,"PacketSent":23,"PacketLost":0,"MessageReceived":2,"MessageSent":2,"LastMessageTime":1648085314,"DisconnectTimes":0,"LostTimes":0}},"time":1648085327}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-24 09:29:02] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 09:29:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 09:29:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 09:29:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-24 09:29:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 1 次重试 
[2022-03-24 09:29:34] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": context deadline exceeded (Client.Timeout exceeded while awaiting headers) 将进行第 2 次重试 
[2022-03-24 09:29:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": read tcp 127.0.0.1:62110->127.0.0.1:8000: wsarecv: An existing connection was forcibly closed by the remote host. 将进行第 3 次重试 
[2022-03-24 09:29:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": read tcp 127.0.0.1:62109->127.0.0.1:8000: wsarecv: An existing connection was forcibly closed by the remote host. 将进行第 2 次重试 
[2022-03-24 09:29:40] [WARNING]: 上报 Event 数据 {"anonymous":null,"font":0,"group_id":822612889,"message":"/luck","message_id":1146403715,"message_seq":287,"message_type":"group","post_type":"message","raw_message":"/luck","self_id":3102833903,"sender":{"age":0,"area":"","card":"","level":"","nickname":"普通人阿文","role":"owner","sex":"unknown","title":"","user_id":1270672343},"sub_type":"normal","time":1648085362,"user_id":1270672343}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-24 09:29:40] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 09:32:29] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 09:32:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 09:32:36] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 09:32:40] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":77,"PacketSent":49,"PacketLost":0,"MessageReceived":12,"MessageSent":9,"LastMessageTime":1648085537,"DisconnectTimes":0,"LostTimes":0}},"time":1648085547}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-24 09:32:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 09:32:45] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 09:32:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 09:32:52] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":77,"PacketSent":49,"PacketLost":0,"MessageReceived":12,"MessageSent":9,"LastMessageTime":1648085537,"DisconnectTimes":0,"LostTimes":0}},"time":1648085560}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-24 09:32:54] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 09:32:58] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 09:33:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 09:33:05] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":78,"PacketSent":50,"PacketLost":0,"MessageReceived":12,"MessageSent":9,"LastMessageTime":1648085537,"DisconnectTimes":0,"LostTimes":0}},"time":1648085572}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-24 09:33:07] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 09:33:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 09:33:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 09:33:18] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":78,"PacketSent":50,"PacketLost":0,"MessageReceived":12,"MessageSent":9,"LastMessageTime":1648085537,"DisconnectTimes":0,"LostTimes":0}},"time":1648085585}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-24 09:33:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 09:33:23] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 09:33:27] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 09:33:30] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":78,"PacketSent":50,"PacketLost":0,"MessageReceived":12,"MessageSent":9,"LastMessageTime":1648085537,"DisconnectTimes":0,"LostTimes":0}},"time":1648085598}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-24 09:33:32] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 09:34:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 09:34:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 09:34:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 09:34:25] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":88,"PacketSent":56,"PacketLost":0,"MessageReceived":14,"MessageSent":11,"LastMessageTime":1648085624,"DisconnectTimes":0,"LostTimes":0}},"time":1648085652}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-24 09:37:12] [ERROR]: Protocol -> unexpected disconnect: EOF 
[2022-03-24 10:11:16] [ERROR]: Protocol -> unexpected disconnect: read tcp 192.168.255.10:65004->49.7.253.244:80: wsarecv: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. 
[2022-03-24 10:19:19] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 10:19:22] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 10:19:26] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 10:22:39] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 10:22:42] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 10:22:46] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 10:22:49] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":315,"PacketSent":213,"PacketLost":0,"MessageReceived":66,"MessageSent":27,"LastMessageTime":1648088448,"DisconnectTimes":2,"LostTimes":0}},"time":1648088557}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-24 10:22:51] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 10:22:55] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 10:22:59] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 10:23:02] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":315,"PacketSent":213,"PacketLost":0,"MessageReceived":66,"MessageSent":27,"LastMessageTime":1648088448,"DisconnectTimes":2,"LostTimes":0}},"time":1648088569}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-24 10:23:04] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 10:23:08] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 10:23:11] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 10:23:15] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":316,"PacketSent":214,"PacketLost":0,"MessageReceived":66,"MessageSent":27,"LastMessageTime":1648088448,"DisconnectTimes":2,"LostTimes":0}},"time":1648088582}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-24 10:23:17] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 10:23:20] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 10:23:24] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 14:29:49] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 14:29:52] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 14:29:56] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 14:29:59] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":1009,"PacketSent":813,"PacketLost":0,"MessageReceived":145,"MessageSent":42,"LastMessageTime":1648103191,"DisconnectTimes":2,"LostTimes":0}},"time":1648103387}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-24 14:30:01] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 14:30:05] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 14:30:09] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
[2022-03-24 14:30:12] [WARNING]: 上报 Event 数据 {"interval":5000,"meta_event_type":"heartbeat","post_type":"meta_event","self_id":3102833903,"status":{"app_enabled":true,"app_good":true,"app_initialized":true,"good":true,"online":true,"plugins_good":null,"stat":{"PacketReceived":1009,"PacketSent":813,"PacketLost":0,"MessageReceived":145,"MessageSent":42,"LastMessageTime":1648103191,"DisconnectTimes":2,"LostTimes":0}},"time":1648103399}
 到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 停止上报：已达重试上线 
[2022-03-24 14:30:14] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 1 次重试 
[2022-03-24 14:30:18] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 2 次重试 
[2022-03-24 14:30:21] [WARNING]: 上报 Event 数据到 http://127.0.0.1:8000/ 失败: Post "http://127.0.0.1:8000/": dial tcp 127.0.0.1:8000: connectex: No connection could be made because the target machine actively refused it. 将进行第 3 次重试 
